### PR TITLE
MANE transcripts report

### LINKED ISSUE #354: MANE transcripts
I put the latest Chanjo2 in the hands of our geneticists and asked them what they would need to put this into clinical routine. It seems close to meet their needs, but they had a bunch of questions on updates to make it fully solve their situation. Let's see what you think - if reasonable or not.

**Describe the solution you'd like**

The geneticists in Lund primarily work with MANE transcripts. Other transcripts are not of interest. (Is it the same in Stockholm?)

On the overview page, a column showing MANE completeness could be shown instead or in addition to the gene column. Would this be possible?

incomplete_genes_overview

Other ideas are welcome as well. Right now, it takes too long time to load the gene specific page (~1 min each), so it would not be feasible for them to click through a bunch of those to check the coverage for their transcript of interest.

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added/Changed/Fixed
- Closes #354 

image

### How to test
- [x]  Tested in a combo with [this Scout branch](https://github.com/Clinical-Genomics/scout/pull/4915), see instructions on that PR

### Expected outcome
- MANE reports should be created using the new endpoint

## Review
- [x] Tests executed by Jakob37, CR
- [x] "Merge and deploy" approved by Jakob37 

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
This one is ready for review now, but I realise that it might behard to test without also a PR in scout.. I'll work on that as well!

File: src/chanjo2/templates/mane-overview.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	<tr>
+        <td class="row">
+            <div class="panel-default">
+				<div class="panel-heading">
+					<strong>Gene {{ gene_id or samples_stats.gene.hgnc_id }}</strong>
+					{% if samples_stats.transcript.mane_select %}
+						<span class="badge">MANE Select: {{samples_stats.transcript.mane_select}}</span>
+					{% endif %}
+					{% if samples_stats.transcript.mane_plus_clinical %}
+						<span class="badge">MANE Plus Clinical:{{samples_stats.transcript.mane_plus_clinical}}</span>
```
Comment (Jakob37) [inline src/chanjo2/templates/mane-overview.html:72]:
```suggestion
						<span class="badge">MANE Plus Clinical: {{samples_stats.transcript.mane_plus_clinical}}</span>
```

File: src/chanjo2/templates/mane-overview.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	<h2>MANE Transcripts coverage report</h2>
+	{% if extras.panel_name %}
+		<p>Based on gene panel: <strong>{{ extras.panel_name }}</strong></p>
+	{% endif %}
+	<br>
+	{% for gene_id, samples_stats in mane_coverage_stats|sort(attribute='0') %}
+	<tr>
+        <td class="row">
+            <div class="panel-default">
+				<div class="panel-heading">
```
Comment (Jakob37) [inline src/chanjo2/templates/mane-overview.html:66]: Code identations seems off here and below

**Review by Jakob37 — APPROVED** [review]
Very nice, thanks 😄 Testing worked well, and only minimal comments on the code.

Looking forward to hear our geneticists thoughts on this. I think it will very helpful for them.

File: src/chanjo2/templates/mane-overview.html (hunk 2 above)
Comment (northwestwitch) [reply src/chanjo2/templates/mane-overview.html:66]: Looks OK on the editor though, weird!

image

I think I'll leave it as it is

**Comment by northwestwitch** [comment]
> Looking forward to hear our geneticists thoughts on this. I think it will very helpful for them.

Cool, thanks for the review. Let me know how it goes! Merging now!

File: src/chanjo2/templates/mane-overview.html (hunk 2 above)
Comment (Jakob37) [reply src/chanjo2/templates/mane-overview.html:66]: Might be mixed tabs and spaces? (Anyway, not a big issue)

**Pull request merged**